TODO.md

```diff
@@ -0,0 +1,3 @@
+- add package and setup for dependancy cruiser?
+- lint-filename: https://ls-lint.org/?
+- unused-exports: https://www.npmjs.com/package/ts-unused-exports ?
```

Comment:
Should we just use issues?

Reply:
Yes,  I can delete this and put it in an issue, thanks. I wasn't sure if we want to do this stuff, it's probably more app specific and not for the monorepo config